ababot/bot/src/commands/mod.rs

```diff
@@ -0,0 +1,5 @@
+// dir_macros::import! {"bot/src/commands"}
```

Comment:
Should we leave unused code as refrence or remove it?

Reply:
We can probably remove it until we find a way to work with it. It's worthless if we don't get intelisens